src/Sql/dbo/Stored Procedures/Collection_ReadByUserId.sql

```diff
@@ -13,7 +13,9 @@ BEGIN
         ExternalId,
         MIN([ReadOnly]) AS [ReadOnly],
         MIN([HidePasswords]) AS [HidePasswords],
-        MAX([Manage]) AS [Manage]
+        MAX([Manage]) AS [Manage],
+        UserDefaultCollectionEmail,
+        Type as [Type]
```

Comment:
Is the alias needed when it's the same?

I also recommend using the square brackets on all column names, e.g. `[Type]`, just to be safe. It doesn't appear to be a reserved word, but it is being syntax highlighted.